extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -91,7 +91,8 @@ export class BitrateManager {
    * @param {!Element} video
    */
   manage(video) {
-    onNontrivialWait(video, () => {
+    video.isBitrateManaged = true;
```

Comment:
I can't make a comment on a line you didn't change, but does MediaPool call `video.changedSources()`?

Reply:
When mediapool generates the `Sources` object it calls `video.changedSources()`.

Check https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/sources.js#L86-L88